ExtraDry/ExtraDry.Core/ExpandoData/ExpandoDataType.cs

```diff
@@ -0,0 +1,54 @@
+﻿namespace ExtraDry.Core;
+
+public enum ExpandoDataType {
```

Comment:
This was imported from what looks like the EF schema.  The summary tags are no longer XML tags and will not generate documents.  Also, we need to justify every data type that is in here, we can't just add them all in case.  At a minimum, I think e-mail address, Url, ImageUrl, need to go.

Reply:
This has been updated as per our conversation.